### ISSUE TITLE
Build without deployment

### ISSUE THREAD

**Issue by dzsquared** [Azure/sql-action#87]
There's a use case where a dacpac might be needed to save to pipeline artifacts or do other deployments - can the action be provided a sqlproj file without connection info to output a dacpac in the pipeline?

**Comment by dzsquared** [comment]
Deprioritizing this from sql-action since it can be accomplished with the shell commands in the pipeline with `dotnet build`

**Comment by johndowns** [comment]
@dzsquared That's disappointing. While it's true that shell commands could be used, isn't that the case for everything? The advantage of using an action is to abstract that away.

**Comment by dzsquared** [comment]
@johndowns agreed on the value of actions to simplify, and will certainly evaluate for future versions.  I should leave this open to better reflect the state of options down the line.
